core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -154,8 +154,8 @@ object StaticFile {
       etagCalc <- etagCalculator(f).map(et => ETag(et))
       res <- Files[F].isFile(f.toPath()).flatMap[Option[Response[F]]] { isFile =>
         if (isFile) {
-
-          if (start >= 0 && end >= start && buffsize > 0) {
+          val requireCondition = start >= 0 && end >= start && buffsize > 0
+          if (!requireCondition) {
```

Comment:
Was this broken before?  This looks correct, but inverted from the previous?

Reply:
It was inverted during the ce3 migration: https://github.com/http4s/http4s/commit/31b54aa0ce46a8450cf94eb1d0ddea7aeeacf6a8#diff-ffd60bd1163f5dd37a78811adfe0b03b0462f98d4c58041c46d11089232fc336L143-L145

From:
```
          require(
            start >= 0 && end >= start && buffsize > 0,
            s"start: $start, end: $end, buffsize: $buffsize")
```
to
```
          if (start >= 0 && end >= start && buffsize > 0) {
            F.raiseError[Option[Response[F]]](
              new IllegalArgumentException(
                s"requirement failed: start: $start, end: $end, buffsize: $buffsize"))
```
which is the opposite.